distribution/correct-sign-path.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="correct-sign-path" default="shorten-gpg-path">
+    <target name="check-keypath-override">
+        <condition property="gpg.keypath.overridden" value="true" else="false">
+            <not>
+                <equals arg1="${gpg.keypath}" arg2="${gpg.default.keypath}"/>
+            </not>
+        </condition>
+
+        <condition property="shorten.gpg.path" value="true" else="false">
+            <and>
+                <isfalse value="${gpg.keypath.overridden}"/>
+                <os family="unix"/>
+            </and>
+        </condition>
+
+        <condition property="copy.gpg.path" value="true" else="false">
+            <and>
+                <isfalse value="${gpg.keypath.overridden}"/>
+                <os family="windows"/>
+            </and>
+        </condition>
+
+    </target>
+
+    <!--
+        Either use a symlink (Unix) to shorten the GPG path or copy the directory (Windows).
```

Comment:
I don't think we even try to sign these on windows. Its worth testing there, but I think we skip the whole thing.

Honestly I'd be totally fine skipping deb and rpm entirely on windows.